### ISSUE TITLE
sync fails on: BadZipFile("File is not a zip file")

### ISSUE THREAD

**Issue by J00MZ** [Azure/azure-cli#9632]
## Describe the bug
Trying to run `sync` command from Linux machine to Blob store according to [documentation](https://docs.microsoft.com/en-us/cli/azure/storage/blob?view=azure-cli-latest#az-storage-blob-sync).  
command fails on zip file error although I'm not uploading any zip file.

**Command Name**
`az storage blob sync`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:


File is not a zip file
Traceback (most recent call last):
  File "/opt/az/lib/python3.6/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 560, in execute
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 618, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 609, in _run_job
    cmd_copy.exception_handler(ex)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/storage/__init__.py", line 240, in new_handler
    handler(ex)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/storage/__init__.py", line 183, in handler
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 588, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 297, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/__init__.py", line 461, in default_command_handler
    return op(**command_args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/storage/operations/azcopy.py", line 36, in storage_blob_sync
    azcopy = _azcopy_blob_client(cmd, client)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/storage/operations/azcopy.py", line 54, in _azcopy_blob_client
    return AzCopy(creds=blob_client_auth_for_azcopy(cmd, client))
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/storage/azcopy/util.py", line 48, in __init__
    _urlretrieve(file_url, install_location)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/storage/azcopy/util.py", line 187, in _urlretrieve
    zip_file = zipfile.ZipFile(io.BytesIO(req.read()))
  File "/opt/az/lib/python3.6/zipfile.py", line 1108, in __init__
    self._RealGetContents()
  File "/opt/az/lib/python3.6/zipfile.py", line 1175, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
```

## To Reproduce:
- `az storage blob sync -c {} -d {} --account-name {} -s {}`

## Expected Behavior
sync suceeds

## Environment Summary
```
Linux-4.15.0-1042-azure-x86_64-with-debian-stretch-sid
Python 3.6.5
Shell: bash
azure-cli: 2.0.66
```

**Comment by J00MZ** [comment]
Oops. 
I see this is exact duplicate of #9525

**Comment by limingu** [comment]
The fix will be in azure-cli: 2.0.67
